specification/cognitiveservices/data-plane/ComputerVision/stable/v2.0/ComputerVision.json

```diff
@@ -1545,6 +1553,33 @@
         "type": "object",
         "format": "file"
       }
+    },
+    "AzureRegion": {
+      "name": "AzureRegion",
+      "description": "Supported Azure regions for Cognitive Services endpoints",
+      "x-ms-parameter-location": "client",
+      "required": true,
+      "type": "string",
+      "in": "path",
+      "x-ms-skip-url-encoding": true,
+      "x-ms-enum": {
+        "name": "AzureRegions",
+        "modelAsString": false
```

Comment:
Please acknowledge that marking `modelAsString` as false may result in breaking changes if new values are introduced in the enum set.

Reply:
Acknowledged, I'm just reverting things back to how they were :).